pkg/virtctl/root.go

```diff
@@ -133,8 +134,36 @@ func Execute() {
 	log.InitializeLogging(programName)
 	cmd, clientConfig := NewVirtctlCommand()
 	if err := cmd.Execute(); err != nil {
-		version.CheckClientServerVersion(&clientConfig, cmd)
-		fmt.Fprintln(cmd.Root().ErrOrStderr(), strings.TrimSpace(err.Error()))
+		if err := CheckClientServerVersion(clientConfig); err != nil {
+			cmd.PrintErrln(err)
+		}
+		cmd.PrintErrln(err)
```

Comment:
Can you use a different name for the error returned by `CheckClientServerVersion`?
I think it will increase readability a lot xD

Reply:
Right! Changed it to `versionErr`.